config/initializers/logging.rb

```diff
@@ -0,0 +1,20 @@
+config = Rails.application.config
```

Comment:
yeah that would help ... might have some warnings etc coming from the
initializers we'd otherwise lose

On Wed, Jun 29, 2016 at 1:34 PM, Jon Moter <notifications@github.com> wrote:

> In config/initializers/logging.rb
> <https://github.com/zendesk/samson/pull/1114#discussion_r69022092>:
>
> > @@ -0,0 +1,20 @@
> > +config = Rails.application.config
>
> Should this file be named something like 00_logging.rb, so it's one of
> the first things loaded?
>
> If any of the other initializers log something, it might go to the wrong
> place.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1114/files/f90c4a9efc377b8d194d909c2dab394f8b75cc6d#r69022092>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ5lBSr3Eh7l6Hnnq9ylMLCva4I9Yks5qQtbugaJpZM4JAxhu>
> .
>